CODEOWNERS

```diff
@@ -0,0 +1,15 @@
+#####################################################
+#
+# List of approvers for this repository
+#
+#####################################################
+#
+# Learn about membership in OpenTelemetry community:
+#  https://github.com/open-telemetry/community/blob/master/community-membership.md
+# 
+#
+# Learn about CODEOWNERS file format: 
+#  https://help.github.com/en/articles/about-code-owners
+#
+
+* @open-telemetry/ruby-approvers
```

Comment:
individual names might work better here. As we [defined the process](https://github.com/open-telemetry/community/blob/master/community-membership.md) of becoming approver as:

> Nominated by a maintainer
>  - With no objections from other maintainers
> - Done through PR to update the CODEOWNER.

For now it will be harder for this repo as you only have a single maintainer. I'm happy to assist to bootstrap the effort, but it will be great to find more people who are "doing ruby" regularly =)